scripts/resetpgctdb.sh

```diff
@@ -0,0 +1,113 @@
+#!/bin/bash
+
+set -e
+
+usage() {
+  cat <<EOF
+$(basename $0) [--force] [--verbose] ...
+All unrecognised arguments will be passed through to the 'psql' command.
+Accepts environment variables:
+- POSTGRESQL_ROOT_USER: A user with sufficient rights to create/reset the CT
+  database (default: root).
+- POSTGRESQL_ROOT_PASSWORD: The password for \$POSTGRESQL_ROOT_USER (default: none).
+- POSTGRESQL_HOST: The hostname of the PostgreSQL server (default: localhost).
+- POSTGRESQL_PORT: The port the PostgreSQL server is listening on (default: 5432).
+- POSTGRESQL_DATABASE: The name to give to the new CT user and database
+  (default: defaultctdb).
+- POSTGRESQL_USER: The name to give to the new CT user (default: cttest).
+- POSTGRESQL_PASSWORD: The password to use for the new CT user
+  (default: beeblebrox).
+- POSTGRESQL_IN_CONTAINER: If set, the script will assume it is running in a Docker
+  container and will exec into the container to operate (default: false).
+- POSTGRESQL_CONTAINER_NAME: The name of the Docker container to exec into (default:
+  pgsql).
+EOF
+}
+
+die() {
+  echo "$*" > /dev/stderr
+  exit 1
+}
+
+collect_vars() {
+  # set unset environment variables to defaults
+  [ -z ${POSTGRESQL_ROOT_USER+x} ] && POSTGRESQL_ROOT_USER="postgres"
+  [ -z ${POSTGRESQL_HOST+x} ] && POSTGRESQL_HOST="localhost"
+  [ -z ${POSTGRESQL_PORT+x} ] && POSTGRESQL_PORT="5432"
+  [ -z ${POSTGRESQL_DATABASE+x} ] && POSTGRESQL_DATABASE="defaultctdb"
+  [ -z ${POSTGRESQL_USER+x} ] && POSTGRESQL_USER="cttest"
+  [ -z ${POSTGRESQL_PASSWORD+x} ] && POSTGRESQL_PASSWORD="beeblebrox"
+  [ -z ${POSTGRESQL_INSECURE+x} ] && POSTGRESQL_INSECURE="true"
```

Comment:
The `POSTGRESQL_INSECURE` argument is missing in the `usage()`. Is that intended?